qa-tests-backend/src/main/groovy/services/CVEService.groovy

```diff
@@ -9,6 +9,10 @@ import io.stackrox.proto.api.v1.ImageCVEServiceGrpc
 import util.Env
 
 class CVEService extends BaseService {
+    static Boolean isPostgresRun() {
```

Comment:
I think it's not needed here as it's defined in `BaseService` as well

Reply:
As far as I can tell, this function is only defined here and in `qa-tests-backend/src/test/groovy/BaseSpecification.groovy`.